### PR TITLE
COBI COOP extension

### PR DESCRIPTION
**Pull request by naik-aakash**
This PR links to #125 

# Todos

- [x] Add tests for analyze and describe
- [x] Fix bugs
- [x] Update documentation
- [x] Update describe module and automatic plots
- [x] Update Interactiveplotter to work for cobi and coop
- [x] update cli
- [x] add cli tests for coop/cobi

### CODE REVIEW

**Comment by naik-aakash** [comment]
Hi @JaGeo, I will hold on to add tests until #102 is merged. I could then reuse test files from there.

Currently, the existing tests are failing as it needs a new release of pymatgen with coop and cobi functionality implemented recently.

**Comment by JaGeo** [comment]
@naik-aakash you could ask for such a release already

**Comment by naik-aakash** [comment]
This PR is ready for review. 😃

**Comment by naik-aakash** [comment]
Should also Close #21 ?

File: lobsterpy/plotting/test/test_plotting.py [hunk 1]
```diff
@@ -13,7 +14,7 @@
 TestDir = CurrentDir / "../../"
 
 
-class InteractiveCohpPlotterTest(unittest.TestCase):
+class PlainInteractiveCohpPlotterTest(unittest.TestCase):
```
Comment (JaGeo) [inline lobsterpy/plotting/test/test_plotting.py:17]: Is this okay? Not sure why it is called Plain now?

**Comment by JaGeo** [comment]
Could you also resolve the conflicts, @naik-aakash ? Thanks!

**Comment by naik-aakash** [comment]
> Could you also resolve the conflicts, @naik-aakash ? Thanks!

Done

**Review by JaGeo — APPROVED** [review]
I have tested the obvious cases. Text, automatic plot and interactive automatic plot reflect the change to COBI, COOP

**Comment by JaGeo** [comment]
once the merge conflicts are solved, I can merge.

**Comment by naik-aakash** [comment]
Thanks @JaGeo , will resolve them as soon as possible today

**Comment by naik-aakash** [comment]
Hi @JaGeo , issues are resolved so this could be merged 😄

**Pull request merged**